### ISSUE TITLE
if pool option is enabled, driver always attempts to connect to localhost

### ISSUE THREAD

**Issue by ccheever** [neumino/rethinkdbdash#82]
For example, this code:
```js
var r = require('rethinkdbdash')(servers: [{host: '52.8.68.118', port: 28015}]);
```

will always generate these errors if Rethink isn't running locally:

```
ReqlDriverError: Failed to connect to localhost:28015
Full error:
{
  "code": "ECONNREFUSED",
  "errno": "ECONNREFUSED",
  "syscall": "connect",
  "address": "127.0.0.1",
  "port": 28015
}.
```

Is there a way to tell the driver to not attempt to connect to localhost without disabling the connection pool?

**Comment by neumino** [comment]
It shouldn't connect to localhost by default. I wrote `servers` in the README, but it's implemented and tested as `hosts`.

I just pushed a fix in 2.0.1.

**Comment by ccheever** [comment]
Cool. Thank you!
